### PR TITLE
parallel get from s3

### PR DESCRIPTION
**Pull request by yingjieMiao**
@leogrim

### CODE REVIEW

File: kifi-backend/modules/cortex/app/com/keepit/cortex/store/VersionedStore.scala [hunk 1]
```diff
@@ -23,6 +23,10 @@ trait VersionedStore[K, M <: StatModel, V <: Versionable[M]] {
   def +=(key: K, version: ModelVersion[M], value: V): this.type
 
   def -=(key: K, version: ModelVersion[M]): this.type
+
+  def batchGet(keys: Seq[K], version: ModelVersion[M]): Seq[Option[V]] = {
+    keys.par.map{ k => get(k, version)}.toArray.toSeq
```
Comment (ymatsuda) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/store/VersionedStore.scala]: Why .toArray.toSeq? How about .seq?

File: kifi-backend/modules/cortex/app/com/keepit/cortex/store/VersionedStore.scala [hunk 2]
```diff
@@ -23,6 +23,10 @@ trait VersionedStore[K, M <: StatModel, V <: Versionable[M]] {
...
+  }
```
Comment (leogrim) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/store/VersionedStore.scala:29]: It looks like our S3 client is blocking, so I'm not sure of the benefits / implications of this keys.par

File: kifi-backend/modules/cortex/app/com/keepit/cortex/store/VersionedStore.scala (hunk 1 above)
Comment (yingjieMiao) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/store/VersionedStore.scala]: tried that first. compiler complained. .seq transfer it to parSeq?

Comment (yingjieMiao) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/store/VersionedStore.scala]: ah. .toSeq and .seq are different.

File: kifi-backend/modules/cortex/app/com/keepit/cortex/store/VersionedStore.scala (hunk 2 above)
Comment (yingjieMiao) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/store/VersionedStore.scala:29]: let's try. BTW, our normalizedURI seqNum bumps up quickly, every two minutes there will be around 100~200 changes. So it gives you the impression that cortex is still ingesting. We can check the actual seqNum from your admin page.

Comment (leogrim) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/store/VersionedStore.scala:29]: @yingjieMiao @stkem @eishay (following on our discussion and the standup meeting) 
That's one of the reasons not to use NormalizedURI's sequence number as the broadcast mechanism here, all the more so as the table is already badly overloaded (cf locking issues with sequence number from yesterday). I still think that Cortex should maintain its own sequence number.

**Pull request merged**